src/Microsoft.DotNet.Darc/DarcLib/Helpers/ProcessExecutionResult.cs

```diff
@@ -22,6 +23,11 @@ public void ThrowIfFailed(string failureMessage)
         }
     }
 
+    public IReadOnlyCollection<string> GetOutput()
```

Comment:
maybe we should be more explicit with the method name? GetOutput doesn't really tell you it's already split by lines

Reply:
Renamed to `GetOutputLines()`.

I think it's better to keep the `\r` too. You could be calling a tool (like `git diff`) on Linux which parses a file with `\r\n` and I guess we'd still not want carriage returns to be in every line.